client/state/user-settings/test/reducer.js

```diff
@@ -19,15 +19,15 @@ import {
 describe( 'reducer', () => {
 	test( 'should export expected reducer keys', () => {
 		expect( Object.keys( reducer( undefined, {} ) ).sort() ).toEqual(
-			[ 'settings', 'unsavedSettings', 'updatingPassword' ].sort()
+			[ 'settings', 'unsavedSettings', 'updatingPassword', 'updating' ].sort()
 		);
 	} );
 
 	describe( 'settings', () => {
 		test( 'should default to a `null` value', () => {
```

Comment:
We should alter this:

```suggestion
		test( 'should default to an empty object', () => {
```